watertap/ui/fsapi.py

```diff
@@ -639,11 +641,14 @@ def __init__(
             (do_export, "export"),
             (do_build, "build"),
             (do_solve, "solve"),
+            (do_initialize, "initialize"),
         ):
             if arg:
                 if not callable(arg):
                     raise TypeError(f"'do_{name}' argument must be callable")
                 self.add_action(getattr(Actions, name), arg)
+            elif name == "initialize":
```

Comment:
this shouldn't be necessary, since if do_initialize is set, the previous 'if' will be activated and add the action

Reply:
@dangunter I threw this in there because we want the initialize function to be optional. In the case that `arg` is `None`, the `else` returns an error, because the other 3 actions are all required. By putting this here we add the initialize function (as `None`) even in the case that it is not provided, rather than erroring out